runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsPlugin.java

```diff
@@ -527,15 +545,21 @@ private void updateConfig()
 
 			this.configCallers = config.callers();
 			relationColorHashMap.put(PlayerRelation.CALLER, config.callerColor());
-			locationHashMap.put(PlayerRelation.CALLER, config.callerHighlightOptions().toArray());
+			if (config.callerHighlightOptions() != null)
+			{
+				locationHashMap.put(PlayerRelation.CALLER, config.callerHighlightOptions().toArray());
+			}
 			getCallerList();
 		}
 
 		this.highlightCallerTargets = config.callersTargets();
 		if (this.highlightCallerTargets)
 		{
 			relationColorHashMap.put(PlayerRelation.CALLER_TARGET, config.callerTargetColor());
-			locationHashMap.put(PlayerRelation.CALLER_TARGET, config.callerTargetHighlightOptions().toArray());
+			if (config.callerTargetHighlightOptions() != null)
+			{
+				locationHashMap.put(PlayerRelation.CALLER_TARGET, config.callerTargetHighlightOptions().toArray());
+			}
```

Comment:
`			if (this.callerTargetHighlightOptions() != null)
			{
				locationHashMap.put(PlayerRelation.CALLER_TARGET, this.callerTargetHighlightOptions.toArray());
			}`

Reply:
As discussed on Discord this will be reworked soon